extensions/arc/src/ui/dashboards/postgres/postgresDiagnoseAndSolveProblemsPage.ts

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as azdata from 'azdata';
+import * as loc from '../../../localizedConstants';
+import { IconPathHelper, cssStyles } from '../../../constants';
+import { DashboardPage } from '../../components/dashboardPage';
+import { PostgresModel } from '../../../models/postgresModel';
+
+export class PostgresDiagnoseAndSolveProblemsPage extends DashboardPage {
+	constructor(protected modelView: azdata.ModelView, private _context: vscode.ExtensionContext, private _postgresModel: PostgresModel) {
+		super(modelView);
+	}
+
+	protected get title(): string {
+		return loc.diagnoseAndSolveProblems;
+	}
+
+	protected get id(): string {
+		return 'postgres-diagnose-and-solve-problems';
+	}
+
+	protected get icon(): { dark: string; light: string; } {
+		return IconPathHelper.wrench;
+	}
+
+	protected get container(): azdata.Component {
+		const root = this.modelView.modelBuilder.divContainer().component();
+		const content = this.modelView.modelBuilder.divContainer().component();
+		root.addItem(content, { CSSStyles: { 'margin': '20px' } });
+
+		content.addItem(this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+			value: loc.diagnoseAndSolveProblems,
+			CSSStyles: { ...cssStyles.title, 'margin-bottom': '20px' }
+		}).component());
+
+		content.addItem(this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+			value: loc.clickTheTroubleshootButton('Postgres'),
+			CSSStyles: { ...cssStyles.text, 'margin-bottom': '20px' }
+		}).component());
+
+		const troubleshootButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			iconPath: IconPathHelper.wrench,
+			label: loc.troubleshoot,
+			width: '160px'
+		}).component();
+
+		troubleshootButton.onDidClick(() => {
+			process.env['POSTGRES_SERVER_NAMESPACE'] = this._postgresModel.namespace();
```

Comment:
We shouldn't be setting environment variables like this IMO. It just pollutes the namespace and makes it more likely issue happens (such as race conditions or unsupported platforms like web). It would be better to pass the additional vars as an option to the command. @chlafreniere @MaddyDev does that seem reasonable?

Reply:
Yeah I don't like it either.  Is there some other way I can pass arguments to a notebook?  Otherwise I can remove these env vars.  If they're not set the notebook will prompt the user to select a server to troubleshoot, so can still work.

Reply:
This gets tricky because it's a book - so theoretically once we add more TSGs we'd need to somehow pass them around between each of the Notebooks which I don't believe is going to be something easy to set up.

@chlafreniere Any ideas here? I'm not thinking of a way we can cleanly handle this so guess we'll just have to do this for now.